client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -129,317 +164,121 @@ class CancelPurchaseForm extends React.Component {
-				value={ this.state.questionOneText }
-				onChange={ this.onTextOneChange }
-				placeholder={ translate( 'What are we missing that you need?' ) }
-			/>
-		);
-		reasons.didNotInclude = (
-			<FormLabel key="didNotInclude">
-				<FormRadio
-					name="didNotInclude"
-					value="didNotInclude"
-					checked={ 'didNotInclude' === this.state.questionOneRadio }
-					onChange={ this.onRadioOneChange }
-				/>
-				<span>{ translate( "This upgrade didn't include what I needed." ) }</span>
-				{ 'didNotInclude' === this.state.questionOneRadio && didNotIncludeInput }
-			</FormLabel>
+		appendRadioOption(
+			'didNotInclude',
+			translate( "This upgrade didn't include what I needed." ),
+			translate( 'What are we missing that you need?' )
 		);
 
-		const onlyNeedFreeInput = (
-			<FormTextInput
-				className="cancel-purchase-form__reason-input"
-				name="onlyNeedFreeInput"
-				id="onlyNeedFreeInput"
-				value={ this.state.questionOneText }
-				onChange={ this.onTextOneChange }
-				placeholder={ translate( 'How can we improve our upgrades?' ) }
-			/>
-		);
-		reasons.onlyNeedFree = (
-			<FormLabel key="onlyNeedFree">
-				<FormRadio
-					name="onlyNeedFree"
-					value="onlyNeedFree"
-					checked={ 'onlyNeedFree' === this.state.questionOneRadio }
-					onChange={ this.onRadioOneChange }
-				/>
-				<span>{ translate( 'The plan was too expensive.' ) }</span>
-				{ 'onlyNeedFree' === this.state.questionOneRadio && onlyNeedFreeInput }
-			</FormLabel>
+		appendRadioOption(
+			'onlyNeedFree',
+			translate( 'The plan was too expensive.' ),
+			translate( 'How can we improve our upgrades?' )
 		);
 
-		const anotherReasonOneInput = (
-			<FormTextInput
-				className="cancel-purchase-form__reason-input"
-				name="anotherReasonOneInput"
-				value={ this.state.questionOneText }
-				onChange={ this.onTextOneChange }
-				id="anotherReasonOneInput"
-			/>
-		);
-		reasons.anotherReasonOne = (
-			<FormLabel key="anotherReasonOne">
-				<FormRadio
-					name="anotherReasonOne"
-					value="anotherReasonOne"
-					checked={ 'anotherReasonOne' === this.state.questionOneRadio }
-					onChange={ this.onRadioOneChange }
-				/>
-				<span>{ translate( 'Another reason…' ) }</span>
-				{ 'anotherReasonOne' === this.state.questionOneRadio && anotherReasonOneInput }
-			</FormLabel>
+		appendRadioOption(
+			'couldNotActivate',
+			translate( 'I was unable to activate or use the product.' ),
+			translate( 'Where did you run into problems?' )
 		);
 
-		// Survey questions only for Jetpack subscriptions
-		const couldNotActivateInput = (
-			<FormTextInput
-				className="cancel-purchase-form__reason-input"
-				name="couldNotActivateInput"
-				id="couldNotActivateInput"
-				value={ this.state.questionOneText }
-				onChange={ this.onTextOneChange }
-				placeholder={ translate( 'Where did you run into problems?' ) }
-			/>
+		appendRadioOption(
+			'anotherReasonOne',
+			translate( 'Another reason…' ),
+			translate( 'How can we improve?' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'How can we improve our upgrades?' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).